spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -90,7 +90,9 @@
         "spark",
         SparkInterpreter.class.getName(),
         new InterpreterPropertyBuilder()
-            .add("spark.app.name", "Zeppelin", "The name of spark application.")
+            .add("spark.app.name",
```

Comment:
Hi Felix,

I started a discussion on this on the user list.  Basically I am creating a zeppelin instance per user and having all instances, by default, use a spark cluster.  When those zeppelin instances all talk to the cluster, it's hard to differentiate which is which.  So I want to start them up with unique names so I can differentiate them.  Asking users to make that change before they start using their instances is error-prone, especially considering I'm trying to introduce them to spark via zeppelin and want them to focus on spark.

Cheers,
Craig

> On Nov 8, 2015, at 14:28, Felix Cheung <notifications@github.com> wrote:
> 
> In spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java:
> 
> > @@ -90,7 +90,9 @@
> >          "spark",
> >          SparkInterpreter.class.getName(),
> >          new InterpreterPropertyBuilder()
> > -            .add("spark.app.name", "Zeppelin", "The name of spark application.")
> > +            .add("spark.app.name",
> Why not set spark.app.name from the interpreter config?
> 
> —
> Reply to this email directly or view it on GitHub.
>